doc/source/reference/c-api.iterator.rst

```diff
@@ -603,15 +602,18 @@ Construction and Destruction
             Indicates that only elements which the operand with
             the ARRAYMASK flag indicates are intended to be modified
             by the iteration. In general, the iterator does not enforce
-            this, it is up to the code doing the iteration to follow
```

Comment:
As long as you are touching this, could you clean up the top line? "Indicates that only elements which the operand with the ARRAYMASK flag indicates are intended to be modified by the iteration" does not make sense